enspara/cluster/kcenters.py

```diff
@@ -85,7 +85,7 @@ def fit(self, X, init_centers=None):
             Begin clustring with these centers as cluster centers.
         """
 
-        t0 = time.clock()
+        t0 = time.perf_counter()()
```

Comment:
Nice catch! Not related to TPT but a good change.

Reply:
I found a number of other instances of this substitution, and accidentally only included this one in this PR. I'm going to remove it here, and submit them all together as a separate PR, but I may have to include them as my python is to recent to use the time.clock function

Reply:
> submit them all together as a separate PR

I love it.